### ISSUE TITLE
Missing config folder in npm package

### ISSUE THREAD

**Issue by neyric** [LinkedDataFragments/Server.js#25]
Since .gitignore contains a line with "config" (which is by the way not consistent with the github repository, since it includes the config folder), npm doesn't include it in the published package.

When you start the ldf-server, you get an error such as :
Error: ENOENT: no such file or directory, open '/Users/myuser/myproject/node_modules/ldf-server/config/config-defaults.json'

Could you remove "config" from the .gitignore and re-publish ? Thx

**Comment by neyric** [comment]
btw, a quick fix is to create the config folder in the node_modules/ldf-server directory, and copy config-defaults.json from the repository.

**Comment by RubenVerborgh** [comment]
Thanks for reporting this issue, it has been fixed and released as v2.0.1.

The config folder is meant to put runtime configuration files (in case you use the server from the repository instead of npm). They are ignored so that they are not accidentally committed. Some configuration files, such as the defaults and examples, are explicitly committed in git (hence the `config` folder while it is ignored). Unfortunately, we did forget about npm following `.gitignore` there.

6c73748de64fe3849c8216666643f3823060d55c now explicitly _excludes_ the defaults from the ignore list, and npm picks this up correctly.
